ement-room.el

```diff
@@ -2419,6 +2419,59 @@ these all require at least version 29 of Emacs):
                                ('remove "removed from"))
                              (ement--format-room space))))))
 
+;;;;; Admin/power commands
+
+(defun ement-room--kick-ban-interactive (prompt)
+  "Interactive arguments for kicking or un/banning a user.
+See `ement-room-kick-user', `ement-room-ban-user', `ement-room-unban-user'.
+Returns (user room session reason)."
+  (ement-room-with-highlighted-event-at (point)
+    (let* ((room ement-room)
+           (event (ewoc-data (ewoc-locate ement-ewoc)))
+           (user (ement-event-sender event)))
+      (if (yes-or-no-p (format "%s user %s <%s> from room %s? "
+                               prompt
+                               (ement-user-displayname user)
+                               (ement-user-id user)
+                               (ement--format-room room)))
+          (list user ement-room ement-session
+                (read-string "Reason (optional): "
+                             nil nil nil 'inherit-input-method))
+        ;; HACK: This isn't really an error, but is there a cleaner way to cancel?
+        (user-error "Aborted")))))
+
+(defun ement-room--kick-ban-user (type user room session reason successfmt)
+  "Issue the API request for kicking or un/banning a user.
+See `ement-room-kick-user', `ement-room-ban-user', `ement-room-unban-user'.
+TYPE is `kick', `ban', or `unban'."
```

Comment:
It would be good to list all the arguments in the docstring, if for no other reason than to silence `checkdoc` in the linting.  :)

Reply:
Done.